reframe/core/pipeline.py

```diff
@@ -609,6 +609,28 @@ def pipeline_hooks(cls):
     #:        }
     #:    }
     #:
+    #: Here are some examples to understand how to set the performance tuple,
+    #: in case the expected reference is a negative or a positive number:
+    #:
+    #: +-------------------------------+--------------+---------------------+----------------------+  # noqa: E501
+    #: | **Performance Tuple**         | **Expected** | **Lowest accepted** | **Highest accepted** |  # noqa: E501
+    #: +-------------------------------+--------------+---------------------+----------------------+  # noqa: E501
+    #: | (100, -0.01, 0.02, 'unit')    | 100 unit     | 99 unit             | 102 unit             |  # noqa: E501
+    #: +-------------------------------+--------------+---------------------+----------------------+  # noqa: E501
+    #: | (100, -0.01, None, 'unit')    | 100 unit     | 99 unit             | inf unit             |  # noqa: E501
+    #: +-------------------------------+--------------+---------------------+----------------------+  # noqa: E501
+    #: | (100, None, 0.02, 'unit')     | 100 unit     | -inf unit           | 102 unit             |  # noqa: E501
+    #: +-------------------------------+--------------+---------------------+----------------------+  # noqa: E501
+    #: | (-100, -0.01, 0.02, 'unit')   | -100 unit    | -101 unit           | -98 unit             |  # noqa: E501
+    #: +-------------------------------+--------------+---------------------+----------------------+  # noqa: E501
+    #: | (-100, -0.01, None, 'unit')   | -100 unit    | -101 unit           | inf unit             |  # noqa: E501
+    #: +-------------------------------+--------------+---------------------+----------------------+  # noqa: E501
+    #: | (-100, None, 0.02, 'unit')    | -100 unit    | -inf unit           | -98 unit             |  # noqa: E501
+    #: +-------------------------------+--------------+---------------------+----------------------+  # noqa: E501
+    #:
+    #: More information in the `assert_reference function
+    #: <deferrable_functions_reference.html#reframe.utility.sanity.assert_reference>`__.  # noqa: E501
+    #:
```

Comment:
This doesn't render correctly for me:

<img width="711" alt="Screenshot 2022-04-05 at 23 09 45" src="https://user-images.githubusercontent.com/611019/161849779-12ac41bd-9a6c-4d31-85bd-c7eb26804d45.png">

Also better try using simple tables instead of grid tables. It should be easier to write them.

Reply:
ah, you are right. It's the noqa comment. I will try to find another way to skip the pep8 error.

Reply:
Also, don't use `unit` as a unit but a real thing, such as `s`, `MB/s`, `ns/day` etc. And perhaps the simple tables could be more compact.

Reply:
I had `MB/s` in the beginning but I wasn't sure what unit would make sense for a `-100` value, temperature in `C`? With simple table it is still more than 90 characters.